go.mod

```diff
@@ -5,12 +5,13 @@ go 1.18
 require (
 	github.com/dustinxie/ecc v0.0.0-20210511000915-959544187564
 	github.com/gofrs/uuid/v5 v5.0.0
+	github.com/google/uuid v1.6.0
```

Comment:
why do we have to uuid packages? I know Illiya aksed to use goohle uuid, but in this case let's remove gofrs completely.

Reply:
removed